### DISCUSSION TITLE
How to change the task status from going to complete

### DISCUSSION THREAD

**Discussion by adarshmadrecha** [madrecha/jamku#615] [QA] [answered]
How to complete a Task?
How to update task status?

**Comment by Divwork92** [comment, marked as answer]
Changing the status from going to completed can also be done in a bulk

1. Select the tasks by clicking on the square box on extreme right
2. Click on 'Update status'
3. Select completed and click on save 

If you want to update the status from going to completed for a single task
1. Click on the pen icon next to clock icon 
2. Select completed and click on update
